index.html

```diff
@@ -363,14 +342,9 @@
         <li>Let <var>doc</var> be the <a>Document</a> of the <a>top level
         browsing context</a>.
         </li>
-        <li>Set the <code>hidden</code> attribute to <code>false</code>.
-        </li>
-        <li>Set the <code>visibilityState</code> attribute to
-        <code>visible</code>.
-        </li>
```

Comment:
We define visibilityState algorithm as "on getting"... it seems rather confusing to "set it". Perhaps I'm overlooking something? The workflow here is: if UA detects a visibility change then it fires the event, the developer gets the event and checks visibilityState which runs the "on getting" algorithm to determine the new visibilityState value.